### DISCUSSION TITLE
How do I clone an external Git repository?

### DISCUSSION THREAD

**Discussion by kaihendry** [community/community#22770] [New to GitHub]
<p>Can I use actions/checkout@v2 to clone down <code>git clone https://go.googlesource.com/image</code>?</p>

**Comment by sr229** [comment, marked as answer]
<p><code>actions/checkout</code> only works on GitHub repositories. Please use the normal <code>git clone</code> instead if you plan to clone the Googlesource repository.</p>

**Comment by kaihendry** [comment]
<p>I am still not 100% familiar with workflow stanzas. Do you have an example?</p>
<p>I do not know how to search for examples using <a href="https://github.com/search" rel="noopener nofollow ugc">https://github.com/search</a></p>
<p>It’s not as easy as … filename:.travis.yml because they could be arbitrary filenames IIUC</p>

**Comment by sr229** [comment]
<p>I think GitHub Staff would like to hear about this, sounds like a UX issue by its own.</p>
